selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java

```diff
@@ -76,6 +76,8 @@ public NewWorkspace(SeleniumWebDriver seleniumWebDriver, ActionsFactory actionsF
     String WORKSPACE_CREATED_DIALOG = "//md-dialog/che-popup[@title='Workspace Is Created']";
     String EDIT_WORKSPACE_DIALOG_BUTTON = "//che-button-primary//span[text()='Edit']";
     String OPEN_IN_IDE_DIALOG_BUTTON = "//che-button-default//span[text()='Open In IDE']";
+    String ORGANIZATIONS_LIST = "//div[@class = 'che-filter-selector-button']";
```

Comment:
This xpath looks to general, IMHO. It's better to add `id` to the WebElement to avoid false triggering.